docs/docs/recipes.md

```diff
@@ -130,7 +130,47 @@ Data sourcing in Gatsby is plugin-driven; Source plugins fetch data from their s
 
 ## Transforming data
 
-Transforming data in Gatsby is also plugin-driven; Transformer plugins take data fetched using source plugins, and process it into something more usable (e.g. JSON into JavaScript objects, markdown to HTML, and more).
+Transforming data in Gatsby is also plugin-driven. Transformer plugins take data fetched using source plugins, and process it into something more usable (e.g. JSON into JavaScript objects, markdown to HTML, and more). One such Transformer plugin is gatsby-transformer-plugin that you can use to transform your markdown files to HTML.
```

Comment:
```suggestion
Transforming data in Gatsby is plugin-driven. Transformer plugins take data fetched using source plugins, and process it into something more usable (e.g. JSON into JavaScript objects, markdown to HTML, and more). `gatsby-transformer-plugin` can transform your markdown files to HTML.
```

I think it would help to make this a little shorter and standalone.